### PR TITLE
Fix Invalid environment variables(NEXT_PUBLIC_BACKEND_URL: Required)

### PR DESCRIPTION
**Pull request by SnoopyDevelops**
https://github.com/reworkd/AgentGPT/issues/508

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -22,6 +22,7 @@ NEXTAUTH_SECRET=$(openssl rand -base64 32)
 ENV="NODE_ENV=development\n\
 NEXTAUTH_SECRET=$NEXTAUTH_SECRET\n\
 NEXTAUTH_URL=http://localhost:3000\n\
+NEXT_PUBLIC_BACKEND_URL=http://localhost:3000\n\
 OPENAI_API_KEY=$OPENAI_API_KEY\n\
```
Comment (asim-shrestha) [inline setup.sh:26]: We should remove the trailing slash. Good otherwise

Comment (SnoopyDevelops) [reply setup.sh:26]: It's not added as the last environment variable.
Still the trailing slash should be removed?

Comment (SnoopyDevelops) [reply setup.sh:26]: @asim-shrestha 
It's inserted before the OPENAI_API_KEY and all other environmental variables have the trailing slash except the last one.
Still the trailing slash should be removed when it's not the last environment variable?

Comment (awtkns) [reply setup.sh:26]: @SnoopyDevelops @asim-shrestha I think the trailing slash is needed here. Looks good to me 🚀

**Review by awtkns — APPROVED** [review]

File: setup.sh (hunk 1 above)
Comment (asim-shrestha) [reply setup.sh:26]: Ooops apologies, didn't realize it was needed for the new line

**Pull request merged**